### ISSUE TITLE
TH - Export report as ppt

### ISSUE THREAD

**Issue by rmroot** [ORNL-AMO/AMO-Tools-Desktop#2990]
See work effort for exporting report as powerpoint.

If doable, implement

**Comment by koay9f** [comment]
Basically one graph or table per slide
Maybe Title slide = Facility Name + "Treasure Hunt"
one slide with table with Annual Energy & cost

**Comment by shivasaurus** [comment]
The work I completed on this is on the branch "issue-2990". Currently it is in a very partial state. The pptxgenjs library is used to create the export. I was able to get data tables to export using the tableToSlides() function of that library. The graphs, however, need to be saved to images when they are drawn/updated in Plotly. Then add the graphs as images to the PowerPoint export. Examples of this can be seen with the Team Summary Pie Chart. The sizing may need to be adjusted based on the image.
